src/Autocomplete/src/Form/AutocompleteChoiceTypeExtension.php

```diff
@@ -27,8 +28,12 @@
  */
 final class AutocompleteChoiceTypeExtension extends AbstractTypeExtension
 {
-    public function __construct(private ?TranslatorInterface $translator = null)
-    {
+    public const CHECKSUM_KEY = '@checksum';
```

Comment:
I'm not 100% of this key beeing hard-coded here and statically accessed in the Controller... but i do not have immediate better solution.

Reply:
I got your point, but in that case I treat the `AutocompleteChoiceTypeExtension` as an information owner. If we want to change the checksum key, we'll have to change only one place. Putting it anywhere else, or creating separate constants won't keep this benefit :/.

Reply:
Ok with that reasoning, thanks!